CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [8.7.0]
+
+
```

Comment:
Just want to get this out of the way so we can review and merge the open pr and start taking up tickets when we want: https://extrahorizon.atlassian.net/projects/EXH/versions/14960/tab/release-report-all-issues.